package.json

```diff
@@ -28,6 +28,7 @@
   },
   "homepage": "https://github.com/thebigredgeek/apollo-resolvers#readme",
   "dependencies": {
+    "assert": "^1.4.1",
```

Comment:
Is there a reason we added `assert` here?  It's built into node's core api

Reply:
We require this, because typescript does not directly access that core api, so while transpiling code from ts -> js, it will give error